tap_github/client.py

```diff
@@ -104,19 +118,23 @@ def get_url_params(
 
         if self.replication_key == "updated_at":
             params["sort"] = "updated"
-            params["direction"] = "asc"
-        elif self.replication_key == "starred_at":
+            params["direction"] = "desc" if self.missing_since_parameter else "asc"
```

Comment:
@ericboucher I think putting `asc` first makes sense, but then I'd change the name of the flag to something like `uses_since_parameter` and make it default to `True`. Wdyt?

Reply:
I think `uses_since_parameter` is a bit confusing since we use it in both cases. It's just that in some cases we can't rely on GitHub to do the work for us :p

Let's keep it like that for now then I think :)